cli/internal/env/env.go

```diff
@@ -9,6 +9,11 @@ import (
 	"github.com/vercel/turborepo/cli/internal/util"
 )
 
+// Prefixes for common system environment variables that we always exclude from prefix
+var blockedEnvVarList = []string{
+	"_VERCEL_",
```

Comment:
Can we read this from somewhere?

e.g.: `TURBO_IGNORE_PREFIX`?

Unfortunately this is _composed_  (e.g. `NEXT_PUBLIC_VERCEL_`) so I'm not sure how to do that gracefully. You have a minute to think through that solution?

Reply:
Good call, going to use `TURBO_CI_VENDOR_ENV_KEY`